src/completion/completeAsl.ts

```diff
@@ -33,7 +33,7 @@ export default function completeAsl(document: TextDocument, position: Position,
 
     const node = findNodeAtLocation(rootNode, offset)
 
-    const snippetsList = completeSnippets(node, offset, aslOptions?.shouldShowStateSnippets)
```

Comment:
If i try to autocomplete within a map state(just an example), i will get duplicated completions for `Retry`/`Catch` with different completion snippets 1 from `snippetsList` the other one from `completionList`, is that expected behavior?

```yaml
    Catch:
      -
        ErrorEquals:
          - States.ALL
        Next: NextState
```
vs
```yaml
    Catch:
      - 
```

Reply:
yes, that is expected. Snippets and completions are different things and have a different icon. Completions are meant to be more basic and allow user to fill out most of the content. Snippets are providing full blown solutions that are only meant to be slightly altered.

Reply:
thanks for explanation.